<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GBA to Pret Music Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>ROM Data</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>65</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Song Table:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NÂº of Songs:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rversion">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>310</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rfile">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>330</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rsongTable">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>295</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0x000000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_rsongNum">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>380</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string>PRET Data</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>65</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Version:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>65</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Song:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pversion">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>300</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pfolder">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>40</y>
       <width>315</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_LastSong">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Config">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QRadioButton" name="radioButton_AName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name Songs Automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_MName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>45</y>
       <width>150</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Name Songs Manually</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Override">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Override Pret Project</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last song</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First song</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_FirstSong">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_LastSong">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_File">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ROM File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PRET Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Extract">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>125</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract Data</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>395</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
